meetings.py

```diff
@@ -39,17 +40,23 @@ def fetch_records(report_url, params, wait=1):
         if 'error' in response.keys():
             raise ZoomApiException(response['error'])
 
-        pages.append(response)
+        [records.append(record) for record in response[listkey]]
+
+        if 'total_records' in response.keys():
```

Comment:
I'm tempted to suggest a `countkey` arg to the function and make the default value 'total_records'. Then you can simply to just:

```
if len(records) >= response[countkey]:
    break
```